tools/indexes-creator/cmd/indexes-creator/main.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/client"
+	"github.com/ElrondNetwork/elastic-indexer-go/client/logging"
+	"github.com/ElrondNetwork/elastic-indexer-go/tools/indexes-creator/reader"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/pelletier/go-toml"
+	"github.com/urfave/cli"
+)
+
+const configFileName = "cluster.toml"
+
+type Config struct {
+	ClusterConfig struct {
+		URL            string   `toml:"url"`
+		Username       string   `toml:"username"`
+		Password       string   `toml:"password"`
+		UseKibana      bool     `toml:"use-kibana"`
+		EnabledIndexes []string `toml:"enabled-indexes"`
+	} `toml:"config"`
+}
+
+var (
+	log = logger.GetOrCreate("main")
+
+	// defines the path to the config folder
+	configPath = cli.StringFlag{
+		Name:  "config-path",
+		Usage: "The path to the config folder",
+		Value: "./config/",
+	}
+)
+
+const helpTemplate = `NAME:
+   {{.Name}} - {{.Usage}}
+USAGE:
+   {{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}
+   {{if len .Authors}}
+AUTHOR:
+   {{range .Authors}}{{ . }}{{end}}
+   {{end}}{{if .Commands}}
+GLOBAL OPTIONS:
+   {{range .VisibleFlags}}{{.}}
+   {{end}}
+VERSION:
+   {{.Version}}
+   {{end}}
+`
+
+func main() {
+	app := cli.NewApp()
+	cli.AppHelpTemplate = helpTemplate
+	app.Name = "Index cr"
+	app.Version = "v1.0.0"
+	app.Usage = "This is the entry point for Elasticsearch reindexing tool"
```

Comment:
`Elasticsearch indices creator tool` *

rename `indexes` to `indices` for directory name, structs, and so on

Reply:
done.